internal/cmd/gocdk/init.go

```diff
@@ -152,15 +152,19 @@ func inferModulePath(ctx context.Context, pctx *processContext, projectDir strin
 	if err != nil {
 		return "", xerrors.Errorf("infer module path: %w", err)
 	}
-	// Check if the projectDir is relative to GOPATH.
-	rel, err := filepath.Rel(strings.TrimSuffix(string(gopath), "\n"), projectDir)
-	if err != nil {
-		return "", xerrors.Errorf("infer module path: %w", err)
```

Comment:
Do your tests cover both this error and the one below? You can check with `codecov`...

Reply:
Not sure if you mean the error on 160 or 168... looks like the former is "no" and the later is "yes".
<img width="884" alt="Screen Shot 2019-06-11 at 11 37 12 AM" src="https://user-images.githubusercontent.com/4240737/59297561-5c3e6f00-8c3d-11e9-991e-b0bab1211610.png">

Reply:
I meant that the tests should try to cover all of the error conditions :-).

Reply:
reprising in-person discussion: not going to add another test for this. can you PTAL, on the PR as a whole?